plugins/dappos/profile.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "dappos",
+  "displayName": "Dappos",
+  "methods": [],
+  "version": "1.0.0-alpha",
+  "url": "https://remix-dappos-plugin.surge.sh/",
+  "description": "Build frontend interfaces for smart contracts with no-code",
```

Comment:
thanks for the PR, could you provide github repo where the code is stored?
we would also probably need to publish the plugin to IPFS, but we can help with that.

Reply:
Thanks @yann300. Haven't deployed the code yet, is that a requirement?

Reply:
repo for plugin: https://github.com/Dappos-Inc/remix-plugin